terraform/sqs/provision/main.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_sqs_queue" "queue" {
+  name       = var.instance_name
+  fifo_queue = false
```

Comment:
praise: thanks for being explicit with this value. It will help in the next iteration